generic_config_updater/generic_config_updater.conf.json

```diff
@@ -48,7 +48,22 @@
         },
         "NTP_SERVER": {
             "services_to_validate": [ "ntp-service" ]
-        }
+        },
+	"PFC_WD": {
+            "table_modification_validators": [ "generic_config_updater.validators.is_mellanox_device_validator" ]
+	},
+	"BUFFER_POOL": {
+            "table_modification_validators": [ "generic_config_updater.validators.is_mellanox_device_validator" ]
+	},
+	"WRED_PROFILE": {
+            "table_modification_validators": [ "generic_config_updater.validators.is_mellanox_device_validator" ]
+	},
+	"QUEUE": {
+            "table_modification_validators": [ "generic_config_updater.validators.is_mellanox_device_validator" ]
+	},
+	"BUFFER_PROFILE": {
+            "table_modification_validators": [ "generic_config_updater.validators.is_mellanox_device_validator" ]
```

Comment:
Offline discussed, this is not flexible to express all the rules: ASIC vendor, ASIC generation, and sonic branches.

Reply:
Previously, the design only allowed for "AND" of all validators
I added a new section to allow for "OR" check of validators
Extending the "OR" section to be a list of lists would allow for (X AND Y AND Z) OR (S AND T) checks as well for even greater flexibility